SharedSupport/Default Bundles/Lorem Ipsum Paragraph.spBundle/command.plist

```diff
@@ -23,17 +23,30 @@ print "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vel mi lacus
 		<key>modifierFlags</key>
 		<integer>393216</integer>
 	</dict>
+	<key>isDefaultBundle</key>
+	<true/>
 	<key>keyEquivalent</key>
 	<string>^$P</string>
+	<key>migratedToNewShortcut</key>
```

Comment:
What if instead of a one-type migration we introduced a version number to bundles and handled bundles without a version number as version 1? We could make these bundles with new-type key equivalents version 2 bundles. That would make it easier in the future if we make bundle changes to introduce a version 3 and beyond

Reply:
OK, let me take a look.